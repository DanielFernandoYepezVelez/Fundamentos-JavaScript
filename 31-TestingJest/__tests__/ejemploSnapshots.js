/* Para Probar El Objeto Completo Yo Puedo Convertir
El Objeto A Un String Con Json.stringify Y Compararlo.
Pero Con Jest Tenemos Otra OpciÃ³n Que Se Conocen Como Los
Snapshots */

/* Se Crea Una Carpeta Aparte Para Compara Los Objetos Completos */
/* Para Actualizar Un Snapshot Ya Existente npm t -- -u */
const cliente = {
    nombre: 'Daniel Fernando',
    balance: 500,
    tipo: 'Premium Gold'
};

describe('Testing Al Objeto De Cliente', () => {
    test('El Cliente Es Daniel', () => {
        expect(cliente).toMatchSnapshot();
    });
});