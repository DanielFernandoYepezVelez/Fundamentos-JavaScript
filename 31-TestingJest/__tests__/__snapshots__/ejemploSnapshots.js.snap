// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Al Objeto De Cliente El Cliente Es Daniel 1`] = `
Object {
  "balance": 500,
  "nombre": "Daniel Fernando",
  "tipo": "Premium Gold",
}
`;
